src/unittests/floating_point.spec.ts

```diff
@@ -106,10 +133,10 @@ interface ConstructorCase {
   expected: IntervalBounds;
 }
 
-g.test('constructor_f32af')
+g.test('constructor_f32f16af')
```

Comment:
I think we probably can drop the type suffixes `_f32f16af`, since there isn't going to be separate tests for different types. These annotations don't add any information, since the `trait` value will show up in the test name/log to differentiate which type is being tested.

Reply:
Done.